strongbox-web-forms/src/main/java/org/carlspring/strongbox/forms/configuration/MavenRepositoryConfigurationForm.java

```diff
@@ -16,6 +21,10 @@
 
     private String cronExpression;
 
+    @DescribableEnumValue(type = MetadataExpirationStrategyType.class,
+            message = "metadataExpirationStrategy must contain one the following strings as value:  checksum, refresh")
```

Comment:
`must contain one the following strings as value`

=>

`must be equal to either: checksum or refresh`